routes/curations.js

```diff
@@ -66,8 +66,7 @@ async function updateCurations(request, response) {
   if (!result.contribution) return response.status(400).send({ errors: result.errors })
   response.status(200).send({
     prNumber: result.contribution.data.number,
-    url: curationService.getCurationUrl(result.contribution.data.number),
-    errors: result.errors
+    url: curationService.getCurationUrl(result.contribution.data.number)
```

Comment:
Since it is either an all or nothing patch now we only return either 200 or 400 
and so we don't care about anything in between in terms of error messages